core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -368,6 +374,12 @@ final class EventHubsConf private (private val connectionStr: String)
     self.get(OperationTimeoutKey) map (str => Duration.parse(str))
   }
 
+  /** Internal operation timeout. */
```

Comment:
Maybe we can explain here that this is _exactly_ the same as `operationTimeout`. The only difference is in the type (one is a Java `Duration` and the other is a Scala `FiniteDuration`)

Reply:
good suggestion. will fix it.